src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -434,10 +450,15 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, OnDestroy
         return (
             keysToCompare.some(key => this.quizExercise[key] !== this.savedEntity[key]) ||
             !this.areDatesIdentical(this.quizExercise.releaseDate, this.savedEntity.releaseDate) ||
+            !this.areCategoriesIdentical(this.quizExercise.categories, this.savedEntity.categories) ||
             !this.areQuizExerciseEntityQuestionsIdentical(this.quizExercise.questions, this.savedEntity.questions)
         );
     }
 
+    areCategoriesIdentical(categoriesUsed: string[], categoriesSaved: string[]): boolean {
```

Comment:
This would return false if the order of categories is changed, is that a what you want?

Otherwise, you should call `.sort()` on the two inputs

Reply:
yeah i thought is doesn't matter that much. it just activates the save button 👍